static/docs/user-guide/install/index.md

```diff
@@ -0,0 +1,62 @@
+# Install
+
+There are a few ways to install DVC: using `pip` or `conda`, which are system
+agnostic, and using packages/installers for specific systems.
```

Comment:
and using -> or using

Reply:
system -> platform (both occurrences)
I would even add a note at the end: "for specific platforms (Linux/Windows/Mac)." to be extra clear what we're referring to.